src/libs/tools/include/helper/cmergehelper.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ *
+ * @brief Helper for merge library
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#ifndef CMERGEHELPER_HPP_
```

Comment:
I really recommend you (again) to avoid using the word "cmerge". It will be a lot of work to rename all of this.

Furthermore, this file actually has nothing to do with merging, I would call it key (or keyset) helpers.

Reply:
There is already a file called keyhelper.hpp and keyhelper.cpp. I now moved the functions into it instead of creating a new file like [initially planned](https://github.com/ElektraInitiative/libelektra/pull/3236#discussion_r347163691).
> It will be a lot of work to rename all of this.

Unfortunately, I don't know how we could completely avoid this. For example, I cannot simply rename `kdb cmerge` to `kdb merge` as it is already in use and it seems like the most appropriate name to me.

Reply:
>  Unfortunately, I don't know how we could completely avoid this. For example, I cannot simply rename kdb cmerge to kdb merge as it is already in use and it seems like the most appropriate name to me.

Of course you can, the old `kdb merge` is to be removed.